docs/source/sctool/task.rst

```diff
@@ -71,6 +65,8 @@ You will have to create a new one.
    sctool task delete -c prod-cluster repair/143d160f-e53c-4890-a9e7-149561376cfd
```

Comment:
@mmatczuk If the command `sctool task <command>` was renamed to `sctool <command>`, we should adapt the code examples manually. 

For the task related commands: Should we keep the examples as they are and update them in a second iteration? Or, would you prefer to delete them for now?

Reply:
There are examples in some of the input files. Use those, others remove for now.